projects/components/CHANGELOG.MD

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0-dev.49] - 2020-11-10
+1. BREAKING: Fixes an error in the type clash with Jasmine and Chai in the Widget Object
+2. Replace DropdownItem with ActionMenuItem
+3. Replace [disabled] with [clrDisabled] on action menu buttons
```

Comment:
The change should not regurgitate the commit message. It should be user friendly explaining what the benefit is. For example, for line 3: Fix arrow navigation of disabled menu items. We should probably ask everyone to modify the changelog as they make changes instead of burdening the person doing the bump